client/src/components/Card.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import '../styles/Card.css';
+
+class Card extends React.Component {
+	constructor(props) {
+		super(props);
+		//this.show_image = this.show_image.bind(this);
+	}
+	
+	render() {
+		return (
+			<div className="card-parent-container">
+			<div className="card-container">
+				<div className="name-mana-container">
+					<div className="name">
+						Sonic Assault	
```

Comment:
At some point, the Card component should eventually rely on using component state for things like name, cost, image, etc. We may also want to separate power and toughness when we do in case we have counters like +1/0 or 0/-2. This can be done by setting the state in the constructor:
```
this.state = {
  name: 'Sonic Assault',
  cost: '{1}{U}{R}',
  image: require("./images/aghoul.jpg"),
  type: 'Instant',
  logo: 00,
  text: 'Tap target creature. Sonic Assault deals 2 damage to that creature\'s controller.\n\nJump-start (You may cast this card from your graveyard by discarding a card in addition to paying its other costs. Then exile this card.)',
  power: 3,
  toughness: 2,
}
```
then at the top of the `render()`, we can use destructuring to pull these variables out of the state:
```
const {
  name,
  cost,
  image,
  type,
  logo,
  text,
  power,
  toughness,
} = this.state;
```
then finally using them
```
<div className="cost">
   {cost}
</div>